BaikalDataGenerator/Source/render.cpp

```diff
@@ -400,38 +402,19 @@ void Render::GenerateDataset(const std::string &path)
         for (auto i = 0u; i < kNumIterations; i++)
         {
             m_renderer->Render(scene);
-        }
 
-        for (const auto& output : outputs_collection)
-        {
-            SaveOutput(output.type,
-                       path,
-                       output.name,
-                       output.file_ext,
-                       output.bpp);
+            if (m_spp.find(i + 1) != m_spp.end())
```

Comment:
It's easier to start iterations from the i = 1

Reply:
Done